frontend/src/features/mapSideBar/tabs/propertyDetails/update/UpdatePropertyDetailsForm.tsx

```diff
@@ -74,24 +74,25 @@ export const UpdatePropertyDetailsForm: React.FunctionComponent<IUpdatePropertyD
   const volumetricMeasurement = getIn(values, 'volumetricMeasurement') as number;
   const volumetricUnit = getIn(values, 'volumetricUnitTypeCode') as string;
 
+  const setFieldValue = formikProps.setFieldValue;
   // clear related fields when volumetric parcel radio changes
   useEffect(() => {
     if (!isVolumetricParcel) {
-      formikProps.setFieldValue('volumetricMeasurement', 0);
-      formikProps.setFieldValue('volumetricUnitTypeCode', undefined);
-      formikProps.setFieldValue('volumetricParcelTypeCode', undefined);
+      setFieldValue('volumetricMeasurement', 0);
+      setFieldValue('volumetricUnitTypeCode', undefined);
+      setFieldValue('volumetricParcelTypeCode', undefined);
     }
-  }, [isVolumetricParcel, formikProps, formikProps.setFieldValue]);
+  }, [isVolumetricParcel, setFieldValue]);
```

Comment:
FYI @FuriousLlama this code was causing infinite render loops, this may have been the root cause of some of your test issues? if any tests got removed/skipped as a result of that this should allow them to be enabled again.